lsstvaultutils/recursivedeleter.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import click
-import hvac
+import hvac  # type: ignore
```

Comment:
Just curious, what error was this producing without the comment?

Reply:
When you lint it with mypy it warns that hvac has no type hints.  

```
lsstvaultutils/recursivedeleter.py:6: error: Skipping analyzing 'hvac': found module but no type hints or library stubs
```